src/query.js

```diff
@@ -0,0 +1,142 @@
+/* Query
+/  
+/  Establish connection with user.js and communicate with teamcity rest api
+/  
+/  Send api requests and pass data to function in main to data.js
+*/
+
+class query {
+
```

Comment:
I guess this comment not only applies to this class, but also some others:
Can we add a constructor here that adds some instance variables, like for example the `tcUser` and and the `teamcity_base_url`. I'm having a hard time figuring out where data is declared and in what classes data is used/modified. I think we can make it more explicit by passing data that this class needs through the constructor, and relying on the instance variables in the methods where possible.